contrib/opencensus-ext-azure/tests/test_azure_heartbeat_metrics.py

```diff
@@ -97,7 +97,7 @@ def test_heartbeat_metric_get_metric_init(self):
         self.assertEqual(metric.NAME, 'Heartbeat')
         keys = list(metric.properties.keys())
         values = list(metric.properties.values())
-        self.assertEqual(len(keys), 2)
+        self.assertEqual(len(keys), 5)
```

Comment:
@lzchen this test was failing on all python versions. Change looks ok?

Reply:
On your local? It's passing for me on local.

Reply:
No, just on the action run. I'll change it back so you can see hold on

Reply:
https://github.com/census-instrumentation/opencensus-python/pull/980/checks?check_run_id=1501556499#step:6:45